include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -140,6 +140,17 @@ namespace openPMD
             log< picLog::INPUT_OUTPUT >( "openPMD: open file: %1%" ) % fullName;
             openPMDSeries = std::unique_ptr<::openPMD::Series >(
                 new ::openPMD::Series( fullName, at, communicator, jsonConfig ) );
+            if( openPMDSeries->backend() == "MPI_ADIOS1" )
+            {
+                throw std::runtime_error(R"END(
```

Comment:
What does `R"END` mean?

Reply:
It's C++11 syntax for multiline strings, see item 6 [here](https://en.cppreference.com/w/cpp/language/string_literal).

Reply:
@franzpoeschel  IS it possible to check in CMake that OpenPMD-api is at least build with ADIOS2 and/or HDF5?

Reply:
Yep, that should be doable. Will get to it.

Reply:
Recent commit does that. I left the runtime check in there.